src/TSID/include/BipedalLocomotion/TSID/QPFixedBaseTSID.h

```diff
@@ -0,0 +1,140 @@
+/**
+ * @file QPFixedBaseTSID.h
+ * @authors Ines Sorrentino
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_QP_FIXED_BASE_TSID_H
+#define BIPEDAL_LOCOMOTION_QP_FIXED_BASE_TSID_H
+
+#include <memory>
+
```

Comment:
```suggestion
#include <optional>

#include <Eigen/Dense>
```

This is the reason why the compilation fails in windows

Reply:
`#include <optional>` was missing

Reply:
also `#include <functional>`